service/model/project.go

```diff
@@ -0,0 +1,91 @@
+package model
+
+import (
+	"context"
+	"time"
+
+	svcerr "release-manager/service/errors"
+
+	"github.com/google/uuid"
+)
+
+type ProjectRepository interface {
+	Create(ctx context.Context, p Project) (Project, error)
+	Read(ctx context.Context, id uuid.UUID) (Project, error)
+	ReadAll(ctx context.Context) ([]Project, error)
+	Update(ctx context.Context, p Project) (Project, error)
+	Delete(ctx context.Context, id uuid.UUID) error
+}
+
+type Project struct {
```

Comment:
Project contains `ReleaseAlertConfig` and `SlackConfig`.

`ReleaseAlertConfig` replaces the original concept of release message templates and determinates which information to include into release alert. I also decided to use `ReleaseAlert` instead of `ReleaseMessage`.

`SlackConfig` has only one field for now - channel id.